docs/modules/ROOT/pages/explanations/pod_security.adoc

```diff
@@ -0,0 +1,151 @@
+https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-auth-pod-security-admission[OpenShift 4.11 also introduces PSA] globally with restricted audit logging and API warnings.
+However, it doesn't replace the existing Security Context Constraints, but runs along side it.
+
+Pod Security Admission was designed to meet the most common security needs out of the box, while being simple to understand and adopt.
+
+At the core of PSA there are the https://kubernetes.io/docs/concepts/security/pod-security-standards/[Pod Security Standards], that define three different policies to broadly cover the security spectrum.
+These policies are cumulative and range from highly-permissive to highly-restrictive.
+
+[cols="1,6"]
+|===
+|*Privileged*
+|Unrestricted policy, providing the widest possible level of permissions.
+This policy allows for known privilege escalations.
+
+|*Baseline*
+|Minimally restrictive policy which prevents known privilege escalations.
+Allows the default (minimally specified) Pod configuration.
+
+|*Restricted*
+|Heavily restricted policy, following current Pod hardening best practices.
+|===
+
+Under Pod Security Admission we understand the built-in Pod Security admission controller to enforce the Pod Security Standards.
+Pod security restrictions are applied at the namespace level when pods are created, and will deny the creation of Pods, Deployments, etc if they don't adhere to the configured Pod Security Standard.
+
+You can configure how PSA will apply for a namespace, by setting specific labels on the namespace.
+These labels define which of the predefined Pod Security Standard levels are used for a namespace. 
+The selected label defines what action the control plane takes if a potential violation is detected:
+
+
+[cols="1,6"]
+|===
+|*enforce*
+|Policy violations will cause the pod to be rejected.
+
+|*audit*
+|Policy violations will trigger the addition of an audit annotation to the event recorded in the audit log, but are otherwise allowed.
+
+|*warn*
+|Policy violations will trigger a user-facing warning, but are otherwise allowed.
+|===
+
+A namespace can configure any or all modes, or even set a different level for different modes.
+Check out https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-namespace-labels/[Enforce Pod Security Standards with Namespace Labels] to see how these can be configured
+
+
+
+.PSA and SCCs
+****
+
+Pod Security Policy and Security Context Constraints run along side each other and OpenShift makes sure that PSA and SCCs are kept in sync.
+
+It does this by looking at the highest privileged SCC available in a namespace and sets the PSA labels accordingly.
+This is handled by the https://github.com/openshift/cluster-policy-controller/tree/master/pkg/psalabelsyncer[Cluster Policy Controller].
+
+So PSA is never more restrictive than SCCs, if a pod is allowed by an SCC it will also be allowed by PSA.
```

Comment:
```suggestion
Since this results in a PSA configuration which is never more restrictive than SCCs, a pod is allowed by an SCC will also be allowed by PSA.
`